hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/symmetric/SecretKeyManager.java

```diff
@@ -109,12 +109,13 @@ public boolean isInitialized() {
    *
    * @return true if rotation actually happens, false if it doesn't.
    */
-  public synchronized boolean checkAndRotate() throws SCMException {
+  public synchronized boolean checkAndRotate(boolean force)
+      throws SCMException {
     // Initialize the state if it's not initialized already.
     checkAndInitialize();
 
     ManagedSecretKey currentKey = state.getCurrentKey();
-    if (shouldRotate(currentKey)) {
+    if (force || shouldRotate(currentKey)) {
```

Comment:
If the rotate is a result of a force, it'll be useful to add a log line to state that.

Reply:
Thanks @duongkame, I have added the log message for the same.